### PR TITLE
Add Pagination component

### PR DESCRIPTION
**Pull request by mkhl**

### CODE REVIEW

File: components/pagination/index.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	}
+	return <nav aria-label={label}>
+		<ul class={klass.join(" ")}>
+			{children}
+		</ul>
+	</nav>;
+}
+
+export function PageItem({ active, disabled, href }, ...children) {
+	let tag ="span";
```
Comment (moonglum) [inline components/pagination/index.jsx]: There is a space missing here, and it is hurting me.

File: components/pagination/index.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	let attrs = { class: "page-link" };
+	let klass = ["page-item"];
+	if(active) {
+		klass.push("active");
+	}
+	if(disabled || !href) {
```
Comment (moonglum) [inline components/pagination/index.jsx:25]: Uh, I like it!

**Review by moonglum — CHANGES_REQUESTED** [review]
Just one change, it is tiny, I promise

File: components/pagination/index.jsx (hunk 1 above)
Comment (mkhl) [reply components/pagination/index.jsx]: Whoops!

**Review by moonglum — APPROVED** [review]
Thanks 🙇

**Pull request merged**